### DISCUSSION TITLE
Possible to reference different Supa projects?

### DISCUSSION THREAD

**Discussion by caseybaggz** [supabase/supabase#29840] [Questions]
Hello all!

I have a unique situation where my company has 2 different projects that serve different purposes (I suppose it's kind of micro-services style?).

DB 1 - responsible for data related to X, has it's own web app
DB 2 - responsible for data related to Y but also references data in X. Different app all together

Is there a world where I would be able to reference a different project within our SB account to achieve DB2 being able to create a foreign key with DB 1 tables? 

It's weird not having that connection but I also am just jumping on the Postgres train (thanks SB!) and understand this is probably abnormal? 🤷‍♀️ 

@supa-alex  or anyone else on the team could maybe give some insight or at least a better strategy within the SB ecosystem?

Thanks!

**Comment by GaryAustin1** [comment]
You should be able to use foreign table wrappers to another postgres database to get a table you can use.
This will not support foreign keys though, but you can join/select the data.
https://www.postgresql.org/docs/current/postgres-fdw.html
https://supabase.com/docs/guides/database/extensions/wrappers/overview

**Reply by caseybaggz** [reply]
@GaryAustin1 thanks so much for that resource! I'll keep you posted if it helps with our design.
